### PR TITLE
Use --quiet flag to reduce maven output (issue #265)

### PR DESCRIPTION
**Pull request by sdelatorrep**

### CODE REVIEW

**Comment by sdelatorrep** [comment]
The paths have changed from `/api/validate` and `/api/debug` to `/validator` and `/validator/debug`
image

**Comment by sdelatorrep** [comment]
OK. Still failing because `/validator/debug` returns some warnings (not errors) and it seems that the previous version returned nothing for the same yaml file:
```
{"schemaValidationMessages":[{"level":"warning","domain":"validation","keyword":"format","message":"format attribute \"uri-reference\" not supported","schema":{"loadingURI":"#","pointer":"/definitions/ExternalDocumentation/properties/url"}},{"level":"warning","domain":"validation","keyword":"format","message":"format attribute \"uri-reference\" not supported","schema":{"loadingURI":"#","pointer":"/definitions/License/properties/url"}}]}
```

**Comment by sdelatorrep** [comment]
BTW, I don't understand why it's returning these 2 warnings as we are following the spec (e.g. https://swagger.io/specification/#licenseObject).

**Comment by teemukataja** [comment]
I tried to fix this as well, by changing `branch 2.0` to `tag v2.0.0` and using the url `/validator?url=` as mentioned in [swagger-validator v2.0.0](https://github.com/swagger-api/validator-badge/tree/v2.0.0#running-locally), but the same issues repeat.

[Here is the full validation error](http://online.swagger.io/validator/debug?url=https://raw.githubusercontent.com/ga4gh-beacon/specification/master/beacon.yaml). There are a lot of missing types and unexpected attributes.

This problem started with the refactoring of `swagger-validator 2.0` 9 days ago, since the last successful Beacon build is from 14 days ago. The [swagger editor](http://editor.swagger.io/?url=https://raw.githubusercontent.com/ga4gh-beacon/specification/master/beacon.yaml) doesn't report any problems, so this might be a bug in the validator, as the two of the errors state "attribute openapi is unexpected" and "attribute swagger is missing", but on the github page of swagger-validator it is stated that openapi is supported as well.

**TL;DR: I suspect that beacon.yaml is intact, and swagger-validator is broken (as of the refactoring on Jan 23rd).**

**Comment by sdelatorrep** [comment]
Closed in favour of #268
